.github/workflows/ci-test.yml

```diff
@@ -311,6 +317,80 @@ jobs:
           $CARGO $CARGOFLAGS test --features='u32' --no-default-features
       - uses: ./.github/actions/build-teardown
 
+  helm-test:
+    runs-on: ubuntu-latest-xl
+    timeout-minutes: 20
+    needs: prepare
+    if: ${{ needs.prepare.outputs.test-helm == 'true' }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - uses: ./.github/actions/check-machine
+        with:
+          min-cpu: 3
+          min-ram: 5
```

Comment:
Since you're using an xl machine now, just delete these lines to prevent slack being warned about machine differences from expect size (the default for the action is the xl machines)